### PR TITLE
vision: move samples from GoogleCloudPlatform/cloud-vision

### PR DESCRIPTION
**Pull request by broady**

### CODE REVIEW

File: vision/label/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Get Dependencies
+
+This sample depends on the following packages. Before getting started, be sure to get them.
+
+```bash
+go get -u golang.org/x/net/context golang.org/x/oauth2/google google.golang.org/api/vision/...
+```
+
+To run tests, getting the following package.
```
Comment (jerjou) [inline vision/label/README.md]: getting -> get

Comment (broady) [inline vision/label/README.md]: Actually I'm removing this whole section about getting dependencies.

We don't do that for any of the other samples.

File: vision/label/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+[cloud-console]: https://console.cloud.google.com
+[vision-api]: https://console.cloud.google.com/apis/api/vision.googleapis.com/overview?project=_
+[adc]: https://cloud.google.com/docs/authentication#developer_workflow
+
+## Run the sample
+
+To build and run the sample:
+
+```bash
+go run label.go ../../data/label/cat.jpg
```
Comment (jerjou) [inline vision/label/README.md]: I think this file doesn't exist in this repo. Perhaps `<your_image_file.jpg>`?

File: vision/label/label.go [hunk 3]
```diff
@@ -0,0 +1,83 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Command label uses the Vision API's label detection capabilities to find a label based on an image's content.
```
Comment (jerjou) [inline vision/label/label.go]: "Command label"? Oh - perhaps "The `label` command..."

File: vision/label/label.go [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+		return err
+	}
+
+	// Parse annotations from responses
+	if annotations := res.Responses[0].LabelAnnotations; len(annotations) > 0 {
+		label := annotations[0].Description
+		fmt.Printf("Found label: %s for %s\n", label, file)
+		return nil
+	}
+	fmt.Printf("Not found label: %s\n", file)
```
Comment (jerjou) [inline vision/label/label.go]: "Not found label" -> "No label found"

File: vision/label/label.go (hunk 3 above)
Comment (broady) [inline vision/label/label.go]: yea, this is the correct style for Go.

File: vision/label/README.md (hunk 2 above)
Comment (broady) [inline vision/label/README.md]: Updated.

**Comment by jerjou** [comment]
LGTM

File: vision/label/label.go [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+	labels, err := findLabels(args[0])
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "%v\n", err)
+		os.Exit(1)
+	}
+	if len(labels) == 0 {
+		fmt.Fprintln(os.Stderr, "No labels found.", err)
+	} else {
+		fmt.Fprintln(os.Stderr, "Found labels:")
```
Comment (jerjou) [inline vision/label/label.go]: Probably want to print to stdout, not stderr

Comment (broady) [inline vision/label/label.go]: Oh, yes

**Comment by jerjou** [comment]
LGTM

**Pull request merged**